lib/src/chart/bar_chart/bar_chart_painter.dart

```diff
@@ -329,7 +329,7 @@ class BarChartPainter extends AxisChartPainter<BarChartData> with TouchHandler<B
             textScaleFactor: textScale);
         tp.layout();
         double x = groupBarPos.groupX;
```

Comment:
Why it doesn't need to be final?

Reply:
Actually, the analyzer didn't warn me about it, so I left it alone.

When I get back to my computer, I'll fix it.

Reply:
Maybe it couldn't be final!

Reply:
You're right, it's actually changed latter!